app/models/post.rb

```diff
@@ -0,0 +1,11 @@
+class Post < ApplicationRecord
+  def publish
+    update(published: true)
+  rescue ActiveRecord::RecordInvalid => e
+  end
+
+  def unpublish
+    update(published: false)
+  rescue ActiveRecord::RecordInvalid => e
+  end
```

Comment:
are we using both the method, i did not find any places, please remove it if we are not using ?